glide-core/src/protobuf/redis_request.proto

```diff
@@ -135,6 +135,7 @@ enum RequestType {
     Rename = 91;
     DBSize = 92;
     Brpop = 93;
+    ZRandMember = 94;
```

Comment:
94 is already busy on upstream by other PR

Reply:
I will fix this when on upstream.